sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosQueryRequestOptions.java

```diff
@@ -59,6 +59,7 @@ public class CosmosQueryRequestOptions {
     private String queryName;
     private CosmosEndToEndOperationLatencyPolicyConfig cosmosEndToEndOperationLatencyPolicyConfig;
     private List<CosmosDiagnostics> cancelledRequestDiagnosticsTracker = new ArrayList<>();
+    private List<String> excludeRegions;
```

Comment:
should we also add in the `CosmosBulkItemRequestOptions`

Reply:
yes please - if patch has its own request options there as well please (but I think it inherits from ItemRequestOptions - so, should get it automatically)

Reply:
done